### DISCUSSION TITLE
0.2.0 Ideas

### DISCUSSION THREAD

**Discussion by wynquinox** [grpc-requests/grpc_requests#85] [General]
Just tossing up some ideas I've had before moving to 0.2.0, as the middle digit bump is a good capture of fundamental changes.

Most of these notes are nebulous at the moment, and are meant to both start a discussion, and get me off my duff to plan out changes.

- Drop official support of 3.7 and 3.8
- Work in 3.9 specific code changes
- Reorganization of client files by class for better readability
- Improving typing
- Move the repo from a personal repo to an organizational repo

**Comment by wynquinox** [comment]
@wesky93 - are you open to assign the repo to an organization that you and I are co-owners on? In general, I think it's best practice for the purposes of making sure we have some continuity on the project just in case either of us moves on from the project - and gives us shared ability to configure secrets, new CI integrations, etc for the project.

I'll volunteer to handle all the set up and responsibility putting it together if you're game.

**Reply by vmesel** [reply]
Just passing by to say that I'm willing to volunteer and help set things up if needed =)

**Reply by wynquinox** [reply]
Thanks @vmesel - I've got the organization set up - just need to collab with @wesky93 as they're the owner of the original repo.

That said, if you're interested in collaborating on the project as we move along - I'm happy to have an extra brain to bounce ideas back and forth with.

**Reply by wesky93** [reply]
@ViridianForge  Thank you so much.
I know a lot of people are going to want to use this project for a long time, so
organisation to use this project for a long time.
If there is anything I should do to do that, please share.

**Reply by wynquinox** [reply]
@wesky93 - I believe you need to follow the instructions [here](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#repository-transfers-and-organizations) and transfer ownership to the [organization](https://github.com/grpc-requests). I sent you an invite to be a co-owner of the organization, so hopefully the transfer ought to be straight forward.

Thank you!

**Reply by wesky93** [reply]
@ViridianForge Done!

**Reply by wynquinox** [reply]
Thanks a ton @wesky93 - much appreciated!
